static/app/components/sidebar/sidebarPanel.tsx

```diff
@@ -69,24 +67,25 @@ function SidebarPanel({
   children,
   ...props
 }: Props): React.ReactElement {
-  const portalEl = useRef<HTMLDivElement>(getSidebarPanelContainer() || makePortal());
+  const portalEl = useRef<HTMLDivElement>(getSidebarPortal());
 
   function panelCloseHandler(evt: MouseEvent) {
     if (!(evt.target instanceof Element)) {
       return;
     }
 
-    const panel = getSidebarPanelContainer();
-
-    if (panel?.contains(evt.target)) {
+    if (portalEl.current.contains(evt.target)) {
       return;
     }
 
     hidePanel();
   }
 
   useEffect(() => {
-    document.addEventListener('click', panelCloseHandler);
+    // Wait one tick to setup the click handler so we don't detect the click
+    // that is bubbling up from opening the panel itself
```

Comment:
Is this enough to ensure we don't detect the click?
I navigate into the onboarding screen through the new card, and then click the back button. After that, clicking the Quick Start sidebar item would hide the panel immediately. Even worse, for the nth time we click on the sidebar item, `hidePanel` would be called 2^n times.

Reply:
Good catch, fix here https://github.com/getsentry/sentry/pull/33769